test/ssl-tests/29-dtls-sctp-label-bug.conf.in

```diff
@@ -0,0 +1,67 @@
+# -*- mode: perl; -*-
+# Copyright 2016-2016 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
2019

Reply:
Fixed in https://github.com/openssl/openssl/pull/7957/commits/7c3860c9a9cf0f6959d06c99cbbabc6fe8197153.

Thanks for reporting.